web/components/FcDataTableRequests.vue

```diff
@@ -136,15 +134,7 @@
       </span>
     </template>
     <template v-slot:item.ASSIGNED_TO="{ item }">
-      <FcMenuStudyRequestsAssignTo
```

Comment:
also this change.  (we can undo it later, but I'd like to really simplify the interface / rendering here - it'll make the whole thing feel much snappier.)